test/cmumps-patient-jsonld-mocha.js

```diff
@@ -20,16 +20,16 @@ describe('chcs-patient-jsonld subgraph', function() {
         });
         server.listen(port);
         return test.createNetwork({
-            chcs: "rdf-components/chcs-patient-jsonld"
+            cmumps: "rdf-components/cmumps-patient-jsonld"
         }).then(function(network){
             var output = noflo.internalSocket.createSocket();
-            network.processes.chcs.component.outPorts.output.attach(output);
+            network.processes.cmumps.component.outPorts.output.attach(output);
             return new Promise(function(done) {
                 output.on('data', done);
-                network.graph.addInitial('localhost:' + port, 'chcs', 'authority');
-                network.graph.addInitial('1000004', 'chcs', 'patient_id');
+                network.graph.addInitial('localhost:' + port, 'cmumps', 'authority');
+                network.graph.addInitial('1000004', 'cmumps', 'patient_id');
             });
-        }).should.eventually.have.property('data', "/patient_graph?dataset=chcs-ab&datatype=all&patientid=1000004");
+        }).should.eventually.have.property('data', "/patient_graph?dataset=cmumps-dataset&datatype=all&patientid=1000004");
```

Comment:
We should talk about this one.  Conor is not renaming his end points and we have no end point that is currently OK with the naming requirement for posting here.   I tried removing the set dataset and setting it to '' but neither worked.  What do you suggest?